shared/actions/tracker.js

```diff
@@ -77,94 +77,67 @@ export function registerTrackerIncomingRpcs (): TrackerActionCreator {
 
 export function getProfile (username: string): TrackerActionCreator {
   return (dispatch, getState) => {
+    const tracker = getState().tracker
+
     // If we have a pending identify no point in firing off another one
-    if (getState().tracker.pendingIdentifies[username]) {
+    if (tracker.pendingIdentifies[username]) {
       console.log('Bailing on simultaneous getProfile', username)
       return
     }
 
+    const uid = tracker.trackers[username] && tracker.trackers[username].userInfo && tracker.trackers[username].userInfo.uid
+    const goodTill = tracker.cachedIdentifies[uid]
+    if (goodTill && goodTill >= Date.now()) {
+      console.log('Bailing on cached getProfile', username, uid)
+      return
+    }
+
     dispatch({type: Constants.updateUsername, payload: {username}})
-    dispatch(triggerIdentify('', username, true, serverCallMap(dispatch, getState, true)))
```

Comment:
i think due to some code changing the triggerIdentify call doesn't actually use this 3rd param